modules/rhcos-enabling-multipath.adoc

```diff
@@ -1,122 +1,65 @@
-01-master-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-kargs-mpath                              52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             105s
-99-master-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-ssh                                                                                 3.2.0             40m
-99-worker-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-worker-ssh                                                                                 3.2.0             40m
-rendered-master-23e785de7587df95a4b517e0647e5ab7   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-rendered-worker-5d596d9293ca3ea80c896a1191735bb1   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+$ coreos-installer install /dev/mapper/mpatha \ <1>
+--append-karg rd.multipath=default \
+--append-karg root=/dev/mapper/dm-mpath-root
 ----
-
-. Check the nodes:
+<1> Indicates the path of the single multipathed device.
 +
-[source,terminal]
-----
-$ oc get nodes
-----
+* For multiple multipath devices, or to be more explicit, it is recommended that the names of the multipath devices are aliased to a World Wide Name (WWN) symlink when you append the kernel arguments , such as `/dev/disk/by-label`. For example:
 +
-.Example output
 [source,terminal]
 ----
-NAME                           STATUS                     ROLES    AGE   VERSION
-ip-10-0-136-161.ec2.internal   Ready                      worker   28m   v1.20.0
-ip-10-0-136-243.ec2.internal   Ready                      master   34m   v1.20.0
-ip-10-0-141-105.ec2.internal   Ready,SchedulingDisabled   worker   28m   v1.20.0
-ip-10-0-142-249.ec2.internal   Ready                      master   34m   v1.20.0
-ip-10-0-153-11.ec2.internal    Ready                      worker   28m   v1.20.0
-ip-10-0-153-150.ec2.internal   Ready                      master   34m   v1.20.0
+$ coreos-installer install /dev/disk/by-id/<wwn_ID> \ <1>
+--append-karg rd.multipath=default \
+--append-karg root=/dev/disk/by-label/dm-mpath-root
 ----
+<1> Indicates the WWN ID of the target multipathed device. For example, `wwn-0xx194e957fcedb4841`.
 +
-You can see that scheduling on each worker node is disabled as the change is being applied.
+Alternatively, you can append the kernel arguments on the kernel command line by using `coreos.inst.install_dev`.
```

Comment:
Should we describe how this is done as an optional step?

Reply:
Yeah, not sure where it belongs. Maybe better to just link to the docs where we detail the "install via coreos.inst kargs" path? E.g. something like:

> This symlink can also be used as the `coreos.inst.install_dev` kernel argument when using the [automated installation procedure](link to relevant page).

?

Reply:
Sure, that sounds good. We currently have technical limitations to our docs that don't allow xrefs from a module such as this, but I have referenced the title of the relevant doc section.